Casamari/MNC017.xml

```diff
@@ -0,0 +1,855 @@
+                              </item>
+                              <item xml:id="q11" n="10">
+                                 <num value="2">፲</num>
+                                 <dim unit="leaf">10</dim>
+                                 <locus from="92r" to="101v"/>
+                              </item>
+                              <item xml:id="q12" n="11">
+                                 <num value="2">፲፩</num>
+                                 <dim unit="leaf">10</dim>
+                                 <locus from="102r" to="111v"/>
+                              </item>
+                              <item xml:id="q13" n="12">
+                                 <num value="2">፲፪</num>
+                                 <dim unit="leaf">10</dim>
+                                 <locus from="112r" to="121v"/>
+                              </item>
+                              <item xml:id="q14" n="13">
+                                 <num value="2">፲፫</num>
+                                 <dim unit="leaf">10</dim>
+                                 <locus from="122r" to="131v"/>
+                              </item>
+                              <item xml:id="q15" n="14">
+                                 <num value="2">፲፬</num>
+                                 <dim unit="leaf">10</dim>
+                                 <locus from="132r" to="141v"/>
+                              </item>
+                              <item xml:id="q16" n="15">
+                                 <num value="2">፲፭</num>
+                                 <dim unit="leaf">10</dim>
+                                 <note>Stub before <locus target="#142"/>.</note>
+                                 <locus from="142r" to="152v"/>
+                              </item>
+                              <item xml:id="q17" n="B">
+                                 <dim unit="leaf">2</dim>
+                                 <locus from="153r" to="154v"/>
+                              </item>
+                           </list>
+                        </collation>
+                        <condition key="intact">
+                           The manuscript is in very good conditions.
+                           Both boards were probably reused, as they show an additional series of board holes on the inner sides of both boards. 
+                           <locus target="#1a"/> is cut and reduced to a fourth of its size. The upper part of <locus from="1a" to="1b"/> are of smaller size, probably because they have been cut. 
+                           The lower of <locus from="152" to="154"/> is of smaller size, probably because they have been cut. 
+                           The sewing has been repaired in correspondence of the center fold of <ref target="#q16"/>. 
+                           Some leaves are slightly stained with water or due to deletion of ink, e.g. on <locus target="#8r #32v #154r"/>. 
```

Comment:
abrasion of ink? We need someone at the end who checks our English :)

Reply:
I also think so. In that case the ink seems washed out rather than erased (abrasion in my mind is connected with the act of erasing). I am also wondering whether it is more appropriate "to deletion" or "to the deletion".